addons/website/static/src/js/tours/tour_utils.js

```diff
@@ -13,6 +13,18 @@ function addMedia(position = "right") {
         run: "click",
     };
 }
+function assertCssVariable(variableName, variableValue, trigger = "iframe body") {
+    return {
+        content: `Check CSS variable ${variableName}=${variableValue}`,
+        trigger: trigger,
+        run: function () {
```

Comment:
```js
run: () => {
```

Reply:
I cannot do that: I access `this` inside the `run` function.